frontend/src/routes/LandingPage/index.jsx

```diff
@@ -118,10 +115,14 @@ const ApplicationDateInfo = ({ admission }) => (
     <p>
       Siste frist for å <StyledSpan bold>legge inn en søknad</StyledSpan> er{" "}
       <StyledSpan bold red>
-        <Moment format="dddd Do MMMM">{admission.public_deadline}</Moment>
+        <FormatTime format="EEEE dd. MMMM">
```

Comment:
`dd.`->`d.`? To be consistent with the rest of the date usages, if you agree with the format we had before